build-aws.json

```diff
@@ -7,49 +7,6 @@
       "image_name":"{{env `IMAGE_NAME`}}"
    },
    "builders":[
-      {
```

Comment:
The build-aws.json file has been reduced to only one region, the default one which currently is `eu-west-1`.